Author: Olivier Sessink <oli4@users.sourceforge.net>
Reviewed-By: Daniel Leidert <dleidert@debian.org>
Origin: http://bluefish.svn.sourceforge.net/viewvc/bluefish?view=revision&revision=7557
Description: Fix a segmentation fault on platforms where G_GSIZE_FORMAT
 does not equal %lu.
Bug: https://bugzilla.gnome.org/679203
Bug-Ubuntu: https://bugs.launchpad.net/bugs/954879
Forwarded: not-needed

--- a/src/file_dialogs.c
+++ b/src/file_dialogs.c
@@ -1409,8 +1409,8 @@
 	tmpstr = g_strdup_printf(_("Filename:%s changed on disk.\n\n"
 							   "Original modification time was %s\n"
 							   "New modification time is %s\n"
-							   "Original size was %lu\n"
-							   "New size is %lu"), filename, oldtimestr, newtimestr, (long unsigned int)oldsize, (long unsigned int)newsize);
+							   "Original size was %"G_GSIZE_FORMAT"\n"
+							   "New size is %"G_GSIZE_FORMAT), filename, oldtimestr, newtimestr, (long unsigned int)oldsize, (long unsigned int)newsize);
 	g_free(newtimestr);
 	g_free(oldtimestr);
 	return tmpstr;
